flutter/lib/benchmark/benchmark.dart

```diff
@@ -233,6 +233,33 @@ class BenchmarkState extends ChangeNotifier {
         datasetsError.map((element) => '\n${++index}) $element').join();
   }
 
+  Future<String> validateOfflineMode(String errorDescription) async {
+    if (!_store.offlineMode) {
+      return '';
+    }
+    final errors = <String>[];
+    for (final job in _getBenchmarkJobs()) {
+      final modelPath = job.benchmark.benchmarkSetting.src;
+      if (isInternetResource(modelPath)) {
+        errors.add(modelPath);
+      }
+      final testDataPath = job.dataset.groundtruthSrc;
+      if (isInternetResource(testDataPath)) {
+        errors.add(testDataPath);
+      }
+      final groundtruthDataPath = job.dataset.path;
```

Comment:
Typo: variables names are switched in lines 246 and 250.

Reply:
Fixed